pkg/utils/container.go

```diff
@@ -163,26 +164,60 @@ func PullImage(image string) error {
 	return nil
 }
 
-//GenEVEImage from docker to outputFile only with defined configDir
-func GenEVEImage(image, outputDir, command, format string, configDir string, size int) (fileName string, err error) {
+//genEVERootFSImage downloads EVE rootfs image from docker to outputDir
+func genEVERootFSImage(image, outputDir string, size int) (fileName string, err error) {
```

Comment:
Let's put it in separate file with other DownloadsFrom (?)

Reply:
@mydatascience done